omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -363,6 +363,14 @@ function PatientSearchWidget(configuration){
                             }
                         });
                     }
+                    else if (column.type === 'location') {
```

Comment:
let's call this 'identifierLocation' instead of 'location' to make it clear that it's the location of the identifier that we are displaying.